sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -558,7 +558,7 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
       ])
       .innerJoin("disbursement.studentAssessment", "studentAssessment")
       .innerJoin("studentAssessment.application", "application")
-      .innerJoin("disbursement.coeDeniedReason", "coeDeniedReason")
+      .leftJoin("disbursement.coeDeniedReason", "coeDeniedReason")
```

Comment:
why did we change this to leftJoin?

Reply:
This was wrongly changed in the previous PR.